UNL-CSE-AMBASSADORS/VITA
# 147 volunteer selecting

Enable a greeter to select which volunteer prepared the taxes for a client.

~**NOTE: DO NOT MERGE**. This is blocked by @andregarivay on the site select ticket.~

**TESTING**
* Run the database refresh script, and the build script.
* Log in and go to the private queue
* Click on any client, you should see a select field above the Finish button. This select should be disabled
* Click the buttons for Check In, Complete Paperwork, and Preparing. The select should now be enabled
* Do not select a volunteer and an alert should appear saying that you must select a volunteer
* Select a volunteer, and hit the Finish button.
* Ensure that in the database, the servicedBy column has the correct id for the volunteer you selected
